cypress/integration/a11y.spec.js

```diff
@@ -88,6 +83,11 @@ describe('Check accessiblity', () => {
                 checkA11y();
             });
 
+            it('Select radio option and check a11y', () => {
+                findByLabelText('e-Apostille service').check();
+                checkA11y();
```

Comment:
worth checking if the button is findable and clickable too

Reply:
Not sure what you mean by `hecking if the button is findable and clickable too`. Are you thinking of checking with an if statement to see if the button exists then executing the code if it does?

Reply:
`cy.findByRole("button").click()`